docs/application/dotnet/guides/nui/text.md

```diff
@@ -342,6 +352,27 @@ The following markup elements are currently supported:
 
 
 
+### Use Markup to represent encoded characters
+
+Markup text is not allowed to contain some characters unless they are representing tags or entities such as "<", ">" or "&". To include these characters as a part of the text, user should use reserved entities like `&lt;`, `&gt;` or `&amp;`, The following example uses reserved entities:

```

Comment:
Suggest rephrasing to:

```suggestion
Markup text is not allowed to contain some characters unless they are representing tags or entities such as "<", ">" or "&". To include these characters as a part of the text, you must use reserved entities such as `&lt;`, `&gt;` , or `&amp;`. The following example uses reserved entities:
```

Reply:
@surbhig12 I edited your comment to remove unneeded space.

Reply:
Thank you